### DISCUSSION TITLE
feat: show start/end of locked range as "half locked"

### DISCUSSION THREAD

**Discussion by marchershey** [easepick/easepick#24] [Q&A]
Ugh.. horrible title. Sorry about that, let me explain. 

The calendar shows disabled days as this:

![image](https://i.imgur.com/phEEpS5.png)

Is there any way to have the start of a locked range look like "half locked," sort of how this is:

![image](https://i.imgur.com/v8kQf4G.png)

See how the beginning and end of a locked range is slanted? Maybe have `lockedStart`, `locked`, and `lockedEnd` css classes? 

If you can get this done by tomorrow, I'll give you $100USD (paypal / venmo). 
You have no idea how bad I need this lol My deadline is tomorrow

I have fallen in love with easypick! It's EXACTLY what I need! Thank you (x100) for creating this!

**Comment by wakirin** [comment]
Contact me via email in my profile.
